src/promptflow/promptflow/_sdk/_pf_client.py

```diff
@@ -54,6 +56,7 @@ def run(
         name: str = None,
         display_name: str = None,
         tags: Dict[str, str] = None,
+        entry: str = None,
```

Comment:
shall we print waring if we specify settigns like variant which does not apply for  eager mode flow?